training/deepspeech_training/util/checkpoints.py

```diff
@@ -1,9 +1,9 @@
 import sys
-import tensorflow as tf
+
 import tensorflow.compat.v1 as tfv1
 
 from .flags import FLAGS
-from .logging import log_info, log_error, log_warn
+from .logging import log_error, log_info, log_warn
```

Comment:
Why change the order here?

Reply:
Autosort?

Reply:
I don't know what the DS policy is for that, I'd have to ask.